pkgs/development/python-modules/importlib-metadata/default.nix

```diff
@@ -22,6 +23,7 @@ buildPythonPackage rec {
   };
 
   nativeBuildInputs = [
+    setuptools
```

Comment:
do you mind adding a comment.

We could also probably add this master if you did a `hostPlatform != targetPlatform` test

Reply:
I'd rather not have those conditionals to keep the difference between cross and not small.  They're also error prone — in this case we'd actually want `hostPlatform != buildPlatform`.

Reply:
sounds good